docs/src/_data/conversions.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * @fileoverview Helper for command conversions
+ * @author Jay-Karia
+ */
+
+"use strict";
+
+//-----------------------------------------------------------------------------
+// Exports
+//-----------------------------------------------------------------------------
+
+module.exports = async function() {
```

Comment:
This isn't necessary. Eleventy can just load the JSON file directly. So you can rename `conversion_map.json` to `conversions.json` and get rid of this file.